cdap-ui/server/config/parser.js

```diff
@@ -35,7 +35,19 @@ var log = log4js.getLogger('default');
 function extractUISettings() {
   try {
     if (require.resolve('./ui-settings.json')) {
-      return require('./ui-settings.json') || {};
+      const uiSettings = require('./ui-settings.json') || {};
+      const uiThemeName = uiSettings['ui.theme'] || 'default';
+      const uiThemePath = `./themes/${uiThemeName}.json`;
+      let uiThemeConfig = {};
+      try {
+        if (require.resolve(uiThemePath)) {
+          uiThemeConfig = require(uiThemePath);
```

Comment:
If this is successful can we log what theme UI is using. Gives some confirmation for the user that the right theme is picked up if they check the logs.